doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -684,11 +684,14 @@ local html = import 'html.libsonnet';
         },
         {
           name: 'manifestJsonEx',
-          params: ['value', 'indent'],
+          params: ['value', 'indent', 'newline', 'key_val_sep'],
```

Comment:
Can you add `availableSince: 'upcoming'`? 

(See `manifestTomlEx`).

Reply:
@sbarzowski Should this be added to `manifestJsonMinified` instead? `manifestJsonEx` is available in the current version and I didn't find a field that indicates parameter list changes only.

Reply:
Yes, yes, that's what I meant.